nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# User can search for local products

Fixes:
- user can now search for products by city and title
-

Changes proposed in this request & reasons behind organizational or architectural decisions:
-
-

Steps to test:
- pull down branch `searchCity`
- in search bar search for product by city or title
- confirm both search terms work
-

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
N/A

Commit message:
- search bar in index method so you can search by city and title

Link to feature ticket:
- #13 

@mighty-mahi-mahi
